paper-ieee-trans.tex

```diff
@@ -57,19 +55,18 @@ \section{Introduction}
 supporting unified access to large--scale computing and intense data analysis
 applications, has been driven by the needs of scientific communities as part of their
 pan--European research collaborations. One such example is the Worldwide
-Large Hadron Collider (LHC) Computing Grid infrastructure, driven
-by the particle physics scientific community, built to satisfy the computing needs
+Large Hadron Collider (LHC) Computing Grid infrastructure: driven
+by the particle physics scientific community; built to satisfy the computing needs
 of processing, storing and analysis of the very large amount of data produced by the
 LHC particle accelerator.
 
 A significant number of innovation projects and technology transfer
-activities -- later discussed in Section~\ref{sec:ev} -- from the European Commission
-(EC)-funded 5th, 6th and 7th Framework Programmes, and recently from the Horizon 2020 programme
-\cite{h2020} had as their main goal to develop software solutions that provide a seamless and
+activities -- later discussed in Section~\ref{sec:ev} -- were funded by the 5th, 6th and 7th Framework Programmes of the European Commission (EC) and recently by the Horizon 2020 programme
+\cite{h2020}. Their main goal was to develop software solutions that provide a seamless and
 transparent access by researchers to distributed computing and data resources.
 
 Major developments carried throughout the years were focused on providing new
-functionalities, whereas in the early days the adoption of software engineering methodologies had a lower priority. 
+functionalities, while at the very beginning the adoption of software engineering processes had a lower priority. 
```

Comment:
-1
there is no need to change methodologies -> process in the whole text. The former one makes sense since we are referring to what we are describing in the text.

Reply:
ok